### PR TITLE
New package: ovmf-*-201911

### PR DESCRIPTION
**Pull request by teldra**
solves #11243

### CODE REVIEW

File: srcpkgs/ovmf/template [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Template file for 'ovmf'
+pkgname=ovmf
+version=stable201903
```
Comment (maxice8) [inline srcpkgs/ovmf/template]: version can be the date, can't it ?

Comment (teldra) [reply srcpkgs/ovmf/template]: @maxice8 You are right, it should work.

File: srcpkgs/ovmf/template [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+license="BSD-2-Clause-Patent, MIT"
+homepage="http://sourceforge.net/apps/mediawiki/tianocore/index.php?title=EDK2"
+distfiles="https://github.com/tianocore/edk2/archive/edk2-stable${version}.tar.gz
+ https://github.com/openssl/openssl/archive/OpenSSL_${_opensslversion//./_}.tar.gz"
+checksum="31c5a5d1579d4e0f15e56beb134020e4e4692713f26bce3d85683fe38a8ca21d
+ 640f3a3c26aef38293b4ab4017562aa7dccf787267382d27fc003538b405bbb5"
+
+case "$XBPS_TARGET_MACHINE" in
+		i686*) ARCH="IA32"; FILE="Ia32";;
+		*) ARCH="X64"; FILE="X64";;
```
Comment (Johnnynator) [inline srcpkgs/ovmf/template]: Shouldn't this package be `i686*` and `x86_64*` only? At least the `cross-compiled` firmware looks like it is always `x86_64`.

Comment (teldra) [reply srcpkgs/ovmf/template]: If it is compiled on i686, it stops compiling with an error, so I made that case. (https://travis-ci.org/void-linux/void-packages/jobs/542221598).
And I don't know, why it should not be compiled on arm or aarch64 for x86_64? On the contrary, I would say that it should be available as i686 and x86_64 firmware. (afaik it is not available as arm target)

Comment (ackalker) [reply srcpkgs/ovmf/template]: Please refer to the section "Preparing Your Development Kit" in the file [BUILDING.txt](https://sourceforge.net/p/refind/code/ci/master/tree/BUILDING.txt) from the rEFInd source code for lots of valuable information on setting up EDK2 (as well as UDK2018) for building. There is information about various quirks and limitations, cross-compiling etc.

**Comment by concatime** [comment]
@teldra Any news since last time?

File: srcpkgs/ovmf/template [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+version=201905
+revision=1
+_opensslversion=1.1.1c
+wrksrc=edk2-edk2-stable${version}
+hostmakedepends="make python3 libuuid-devel bc acpica-utils nasm"
+makedepends="perl-LWP"
+short_desc="Tianocore UEFI firmware for qemu"
+maintainer="teldra <teldra@rotce.de>"
+license="BSD-2-Clause-Patent, MIT"
+homepage="http://sourceforge.net/apps/mediawiki/tianocore/index.php?title=EDK2"
```
Comment (abenson) [inline srcpkgs/ovmf/template]: This link doesn't work for me.

File: srcpkgs/ovmf/template [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# Template file for 'ovmf'
+pkgname=ovmf
+version=201905
```
Comment (abenson) [inline srcpkgs/ovmf/template]: 201908 is out

**Comment by teldra** [comment]
archs="~i686" at the moment, because it fails building it: https://travis-ci.org/void-linux/void-packages/jobs/604188526?utm_medium=notification&utm_source=github_status

File: srcpkgs/ovmf/template (hunk 3 above)
Comment (teldra) [reply srcpkgs/ovmf/template]: Changed link to tianocore homepage.

File: srcpkgs/ovmf/template (hunk 2 above)
Comment (teldra) [reply srcpkgs/ovmf/template]: The template now supports building the targets X64(x86_64) and IA32(i686) on all void archs. On a i686 host,it builds only IA32.

File: srcpkgs/ovmf/template (hunk 4 above)
Comment (teldra) [reply srcpkgs/ovmf/template]: Implemented.

**Pull request closed without merging**